tests/ui/layout/debug.rs

```diff
@@ -1,4 +1,5 @@
 //@ normalize-stderr-test: "pref: Align\([1-8] bytes\)" -> "pref: $$SOME_ALIGN"
+//@ normalize-stderr-test: "randomization_seed: \d+" -> "randomization_seed: $$SEED"
```

Comment:
Question: is it possible for `-Zrandomize-layout` to take some kind of initial seed? This being normalized out makes me slightly nervous for reproducibility.

Reply:
The randomization seed for individual types must depend on the DefPathHash for said type to ensure that different types with identical declaration are randomized differently. The DefPathHash depends on StableCrateId which in turn depends on the exact version of rustc including commit hash.